float32 heading
float32 roll
float32 pitch
float32 temperature
float32 depth
float32 m
float32 mx
float32 my
float32 mz
float32 a
float32 ax
float32 ay
float32 az
float32 angular_velocity_x
float32 angular_velocity_y
float32 angular_velocity_z
float32 depth_filt
float32 depth_der
float32 pitch_filt
float32 pitch_der
